python/tvm/testing.py

```diff
@@ -795,15 +795,32 @@ def _auto_parametrize_target(metafunc):
 
     """
     if "target" in metafunc.fixturenames:
-        parametrized_args = [
-            arg.strip()
-            for mark in metafunc.definition.iter_markers("parametrize")
-            for arg in mark.args[0].split(",")
-        ]
-
-        if "target" not in parametrized_args:
-            # Check if the function is marked with either excluded or
-            # known failing targets.
+        for mark in metafunc.definition.iter_markers("parametrize"):
```

Comment:
nit: Better not to use loop-else construct (e.g., https://intoli.com/blog/for-else-in-python/).

Reply:
No problem, and I can change that.  The loop-else reads as cleaner to me, even in the blog post's example, but having a consistent style with the rest of the TVM codebase is far more important so I'll avoid using them.